near/package.json

```diff
@@ -21,6 +21,7 @@
     "env-cmd": "^10.1.0",
     "ethers": "^5.6.5",
     "near-api-js": "^0.43.1",
+    "near-seed-phrase": "^0.2.0",
```

Comment:
Should the near-api-js version be bumped up to 1.0.0?  Or it doesn't matter because this is test?

Reply:
Don't think it matters, because they're just test scripts and the near-api-js 1.0.0 didn't change much